src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
It's recommended not to use tag names for styling, except for `html` and `body`. However, since `body` is a unique element and commonly used to reset default margins, this is acceptable.